build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```

Comment:
I have to say I would have preferred that changes like this to the build configuration are kept separate from PRs for other features.